modules/cluster-and-ha-management/pages/set-up-crr.adoc

```diff
@@ -80,6 +67,14 @@ $ gadmin config apply -y
 ====
 These steps above can be skipped if these settings are already configured properly.
 ====
++
+[NOTE]
+====
+About `System.CrossRegionReplication.TopicPrefix`.
+We have a set of internal topics that serve GPE/GSQL etc., e.g. `GSE_journa_*`, `Metadata`. Whenever we start CRR, the data synchronization component creates replicas of these internal topics on the DR cluster. These replica topics are prefixed with "primary." to the original topic name.
+So at this point the DR cluster has at least two sets of internal topics, one initialized during installation and one created by CRR synchronization. It is through this configuration `System.CrossRegionReplication.TopicPrefix` that the DR cluster decides which set of internal topics to use.
+If the situation becomes more complicated, suppose this DR cluster is switched to a PR cluster, this configuration must remain unchanged. When we need to setup a new DR cluster for this cluster, the CRR component will add the "primary." prefix again when it creates a replica of the topics, so this configuration needs to be set to "primary.primary." for the new DR cluster.
+====
```

Comment:
It might be too complicated for the users to understand. 
* I think we'd better put in the section "Fail over to the DR cluster" or `CRR FAQ`
* If possible, we can draw a picture to help ppl understand.

@victorleeTG any insights on improving this?

Reply:
yeah, I realized that the [failover](https://docs.tigergraph.com/tigergraph-server/current/cluster-and-ha-management/fail-over#_set_up_a_new_dr_cluster_after_failover) page was already clear on this concept, so I added the corresponding link directly to L53

Reply:
@pingxieTG @fanwuTG  - This looks complicated and can be simplified with an example for better readability.

Note: The System.CrossRegionReplication.TopicPrefix configuration is critical for managing internal topics during data synchronization. When CRR is enabled, the data synchronization component creates replicas of internal topics on the DR cluster. These replicas are prefixed with primary. followed by the original topic name.

For example, if the DR cluster is later switched to a primary cluster, the System.CrossRegionReplication.TopicPrefix configuration must remain unchanged to ensure consistency. If a new DR cluster is set up for this cluster, CRR will add the primary. prefix again. In such cases, the prefix might need to be configured as primary.primary. to distinguish between different sets of topics.

Reply:
@arun-TG-PM yes, this part has already been simplified in the latest commit.(This comment is outdated)